build.gradle

```diff
@@ -49,7 +49,7 @@ configurations {
 
 dependencies {
 	enigmaRuntime "net.fabricmc:stitch:0.4.3.71"
```

Comment:
I would put this in a separate PR, as ideally other deps need updating as well for the asm 8 move

Reply:
reverted enigma update